kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -178,19 +161,20 @@ deploy_kasfleetmanager() {
     -p MAS_SSO_BASE_URL="${MAS_SSO_BASE_URL}" \
     -p MAS_SSO_REALM="${MAS_SSO_REALM}" \
     -p OSD_IDP_MAS_SSO_REALM="${OSD_IDP_MAS_SSO_REALM}" \
-    -p ENABLE_READY_DATA_PLANE_CLUSTERS_RECONCILE="false" \
+    -p ENABLE_READY_DATA_PLANE_CLUSTERS_RECONCILE="${ENABLE_READY_DATA_PLANE_CLUSTERS_RECONCILE}" \
+    -p SERVICE_PUBLIC_HOST_URL="https://kas-fleet-manager-${KAS_FLEET_MANAGER_NAMESPACE}.apps.${K8S_CLUSTER_DOMAIN}" \
     -p DATAPLANE_CLUSTER_SCALING_TYPE="manual" \
     -p CLUSTER_LIST='
-- "name": "'${DATA_PLANE_CLUSTER_CLUSTER_ID}'"
+- "name": "'$(${OC} config view --minify --raw | yq e '.contexts[0].name' -)'"
+  "provider_type": "'${PROVIDER_TYPE}'"
   "cluster_id": "'${DATA_PLANE_CLUSTER_CLUSTER_ID}'"
-  "client_id": "'${MAS_SSO_DATA_PLANE_CLUSTER_CLIENT_ID}'"
```

Comment:
Good catch! This was also never set/used by KFM as it is was not a valid config entry.